inference-engine/thirdparty/clDNN/src/normalize.cpp

```diff
@@ -29,7 +29,17 @@ primitive_type_id normalize::type_id() {
 layout normalize_inst::calc_output_layout(normalize_node const& node) {
     assert(static_cast<bool>(node.get_primitive()->output_data_type) == false &&
            "Output data type forcing is not supported for normalize_node!");
-    return node.input().get_non_padded_output_layout();
+    auto input_node_layout = node.input().get_non_padded_output_layout();
+    auto output_type =
+        node.get_primitive()->output_data_type ? *node.get_primitive()->output_data_type : input_node_layout.data_type;
```

Comment:
`output_data_type` can't be used here, because of the assertion above. Just use input data type as default out precision. and overwrite it if a node has fused ops.